docs/_panels/jenkins.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Jenkins
+description: Information about Jenkins data
+author: Bitergia
+screenshot: sigils/jenkins.png
+created_at: 2020-03-11
+grimoirelab_version: 0.2.38
+layout: panel
+---
+
+This panel provides an overview of the data extracted from Jenkins instances.
+
+## Metrics
+
+From left to right and top to bottom, the metrics provided are:
+
+* **Summary**: total numbers of builds, jobs, nodes and build durations.
+* **Results**: a pie chart that summarizes the proportions of the build results (success, failure, unstable, aborted).
+* **Nodes**: a table that details the number of builds and durations per node.
+* **Builds over time**: a bar chart that shows the evolution of builds over time.
+* **Active nodes over time**: a bar chart that shows the evolution of active nodes over time.
+* **Builds**: a table that provides details about the single builds.
+* **Jenkins projects**: a pie chart that highlights the builds by project.
```

Comment:
We should specify what we are counting in the pie chart (builds) in the visualization metric label and in the widget title that could be `Jenkins Projects and Builds`